src/main/java/mocacong/server/service/RefreshTokenService.java

```diff
@@ -0,0 +1,52 @@
+package mocacong.server.service;
+
+import lombok.RequiredArgsConstructor;
+import mocacong.server.domain.Member;
+import mocacong.server.domain.Token;
+import mocacong.server.exception.notfound.NotFoundMemberException;
+import mocacong.server.exception.unauthorized.InvalidRefreshTokenException;
+import mocacong.server.repository.MemberRepository;
+import org.springframework.data.redis.core.RedisTemplate;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
+@Service
+@RequiredArgsConstructor
+public class RefreshTokenService {
+
+    private final MemberRepository memberRepository;
+    private final RedisTemplate<String, Token> redisTemplate;
+
+    @Transactional
+    public void saveTokenInfo(Long memberId, String refreshToken, String accessToken) {
+        Token token = Token.builder()
+                .id(memberId)
+                .refreshToken(refreshToken)
+                .accessToken(accessToken)
+                .expiration(1728000) // 리프레시 토큰 유효기간
+                .build();
+
+        redisTemplate.opsForValue().set(refreshToken, token, 1728000, TimeUnit.SECONDS);
+    }
+
+    public Member getMemberFromRefreshToken(String refreshToken) {
+        Token token = redisTemplate.opsForValue().get(refreshToken);
+        if (token != null && token.getExpiration() > 0) {
```

Comment:
null 처리 좋아요~
49번째 라인에 있는, 메리가 만들어준 메서드를 이용할 수도 있어보이네요 
```suggestion
        Token token = findTokenByRefreshToken(refreshToken);
        if (token != null && token.getExpiration() > 0) {
```